### DISCUSSION TITLE
How to delete a key ?

### DISCUSSION THREAD

**Discussion by MaitreGEEK** [tolgee/tolgee-platform#2456] [Q&A] [answered]
Hello, I would like to remove a key but I don't know how to do it. Thanks

**Comment by JanCizmar** [comment, marked as answer]
Hello! 

We probably should have this documented. Huh!

1. Go to the translations view
2. Click the checkbox on the above the key and translations
3. Select delete keys in the batch operation selection dropdown.

<img width="800" alt="image" src="https://github.com/user-attachments/assets/8a02fa23-115c-4446-aed3-813b1acc07ba">
